neo/Prompt/Commands/Send.py

```diff
@@ -31,6 +31,10 @@ def construct_and_send(prompter, wallet, arguments, prompt_password=True):
         address_to = get_arg(arguments, 1)
         amount = get_arg(arguments, 2)
 
+        if int(float(amount)) == 0:
+            print("invalid amount format")
```

Comment:
can we make the message more direct like; "amount cannot be 0" or something along those lines. As this message might suggest "0" is not valid and should be 0.0 or something.

Reply:
Absolutely. Thank you for the feedback!